cardano-node/app/cardano-node.hs

```diff
@@ -73,164 +57,26 @@ main = do
         <$$> ""
         <$$> parserHelpOptions nodeCliParser
 
-      cliHelpTracing :: String
-      cliHelpTracing = renderHelpDoc 80 $
-        "Additional tracing options:"
-        <$$> ""
-        <$$> parserHelpOptions cliTracingParser
 
-      cliHelpAdvanced :: String
-      cliHelpAdvanced = renderHelpDoc 80 $
-        "Advanced options:"
-        <$$> ""
-        <$$> parserHelpOptions parseCommonCLIAdvanced
```

Comment:
So, are we regressing the help messages here?  I think we discussed this previously.

Reply:
Thanks for catching this.

Reply:
This is the reference output: https://github.com/input-output-hk/cardano-node/pull/182#issuecomment-533640238